cdbot/cogs/quoting.py

```diff
@@ -0,0 +1,382 @@
+from math import ceil
+from typing import Callable, List, Union
+
+from discord import Colour, Embed, HTTPException, Member, Message, NotFound, RawReactionActionEvent, TextChannel, User
+from discord.ext import commands
+from discord.ext.commands import ArgumentParsingError, Bot, CheckFailure, Cog, UserConverter
+from motor.motor_asyncio import AsyncIOMotorClient
+
+from cdbot.constants import (
+    CYBERDISC_ICON_URL, LOGGING_CHANNEL_ID, MongoDB, QUOTES_CHANNEL_ID, QUOTES_DELETION_QUOTA, QUOTE_CZAR_ID, SERVER_ID
+)
+
+
+def is_quote_czar() -> Callable:
```

Comment:
I'd have thought this should be something more like `user_can_quote()`, as it checks for Quote Czar (which could be renamed) and admin?

Reply:
Yeah, I'll rename it so it's clearer